### PR TITLE
Remove system channel from Test Network tutorial

### PR DESCRIPTION
**Pull request by pamandrejko**
Signed-off-by: pama-ibm <pama@ibm.com>

- Removed content that described the system channel
- Removed content that discussed consortiums
- Updated command output based on actual output from the sample
- Mentioned `osnadmin` command in the details about the createChannel sub command.
- Removed troubleshooting topic that referenced the system channel.

#### Type of change

- Documentation update

#### Description

Test network tutorial needs to match the updated Fabric Sample test-network.

### CODE REVIEW

File: docs/source/test_network.md [hunk 1]
```diff
@@ -539,11 +531,8 @@ below provide a guided tour of what happens when you issue the command of
 
 - If you use the `createChannel` subcommand, `./network.sh` runs the
   `createChannel.sh` script in the `scripts` folder to create a channel
-  using the supplied channel name. The script uses the `configtx.yaml` file to
-  create the channel creation transaction, as well as two anchor peer update
-  transactions. The script uses the peer cli to create the channel, join
-  ``peer0.org1.example.com`` and ``peer0.org2.example.com`` to the channel, and
-  make both of the peers anchor peers.
+  using the supplied channel name. The script uses the `configtxgen` tool to create the channel genesis block
+  based on the `TwoOrgsApplicationGenesis` channel profile in the `configtx/configtx.yaml` file. If you do not specify a channel name, the genesis block is named `mychannel.block` and is stored in the `channel-artifacts` folder. The  genesis block is then passed to the `osnadmin` command, which creates the channel. Finally, the script uses the peer cli to join ``peer0.org1.example.com`` and ``peer0.org2.example.com`` to the channel, and make both of the peers anchor peers.
```
Comment (nikhil550) [inline docs/source/test_network.md]: This strikes me like a lot of information for this basic tutorial. I think the details of how the channel creation works can be handles in later tutorials, and the profile name and different tools can be discussed later.

**Review by nikhil550 — COMMENTED** [review]
LGTM, Just had one small nit.

File: docs/source/test_network.md (hunk 1 above)
Comment (pamandrejko) [reply docs/source/test_network.md]: Per our discussion on our workgroup call this morning I simplified this description.

**Review by denyeart — APPROVED** [review]

**Comment by pamandrejko** [comment]
@Mergifyio backport release-2.3

**Comment by pamandrejko** [comment]
@Mergifyio backport release-2.2

**Pull request merged**